app/src/main/java/com/belatrixsf/allstars/ui/account/AccountFragment.java

```diff
@@ -208,7 +211,35 @@ public void showSkypeId(String skypeID) {
 
     @Override
     public void showProfilePicture(final String profilePicture) {
-        Glide.with(getActivity()).load(profilePicture).fitCenter().transform(new BorderedCircleTransformation(getActivity())).into(pictureImageView);
+        Glide.with(getActivity())
+                .load(profilePicture)
+                .fitCenter()
+                .transform(new BorderedCircleTransformation(getActivity()))
+                .listener(new RequestListener<String, GlideDrawable>() {
+                    @Override
+                    public boolean onException(Exception e, String model, Target<GlideDrawable> target, boolean isFirstResource) {
+                        startPostponedEnterTransition();
+                        return false;
+                    }
+
+                    @Override
+                    public boolean onResourceReady(GlideDrawable resource, String model, Target<GlideDrawable> target, boolean isFromMemoryCache, boolean isFirstResource) {
+                        startPostponedEnterTransition();
+                        return false;
+                    }
+                })
+                .into(pictureImageView);
```

Comment:
I did not see that piece of code. I am going to add the implementation in the loader.